libs/dextool/source/dextool/compilation_db/package.d

```diff
@@ -739,19 +764,31 @@ ParseFlags parseFlag(CompileCommand cmd, const CompileCommandFilter flag_filter,
                 st = State.keep;
             } else if (st == State.pathArgumentToAbsolute) {
                 st = State.keep;
-                auto p = buildNormalizedPath(workdir, arg).absolutePath;
-                rval.put(p);
-                includes.put(p);
+                putNormalizedAbsolute(arg);
             } else if (st == State.priorityKeepNextArg) {
                 st = State.keep;
                 rval.put(arg);
+            } else if (st == State.checkingForQuotation) {
+                if (!path.data.empty) {
+                    path.put(" ");
+                }
+                path.put(arg);
+                if (isEndingWithQuotation(arg)) {
+                    st = State.keep;
+                    putNormalizedAbsolute(path.data.join.strip('"'));
+                    path.clear();
+                }
             } else if (excludeStartWith(arg, flag_filter)) {
                 st = State.skipIfNotFlag;
+            } else if (isSpacedInclude(arg)) {
+                st = State.checkingForQuotation;
+                rval.put("-I");
+                if (!arg[2 .. $].empty) {
+                    path.put(arg[2 .. $]);
```

Comment:
This most probably should be `arg[3 .. $]` because arg is in this case `-I"foo bar`. This, I think, will be shown when the test case is adjusted to contain a quotation mark